### PR TITLE
Remove census schema files and add business and social files

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?

This PR removes the census schema files and adds business and social files

### How to review

Check changes make sense and reflect what is on the [trello card](https://trello.com/c/tBK7UtE0/4956-eq-questionnaire-schemas-remove-census-schema-files-and-add-business-and-social-files-1-sp)

### CODE REVIEW

**Review by MebinAbraham — COMMENTED** [review]
- The `jsonnet` source files are still there but needs to be removed.
- The README text doesn't quite make sense, as it still refers to building a schema etc. This repo is just a registry now.
- I know the card says `schema` but I think the `schema` dir should be renamed to `schemas` since it will contain multiple schemas. That is also what it used to be.

**Review by LJBabbage — CHANGES_REQUESTED** [review]
My schemas folder doesn't look how I was expecting, I have this 
image
I was thinking there was going to a business and social folder with all the census schemas removed

**Comment by LJBabbage** [comment]
> My schemas folder doesn't look how I was expecting, I have this

Ok, I see why there isn't a schemas folder now, but shouldn't there be?

**Comment by rmccar** [comment]
> > My schemas folder doesn't look how I was expecting, I have this
> 
> Ok, I see why there isn't a schemas folder now, but shouldn't there be?

Looks like the folder hasn’t been checked in because its being ignored even though it isn't listed in gitignore

**Comment by rmccar** [comment]
> > > My schemas folder doesn't look how I was expecting, I have this
> > 
> > 
> > Ok, I see why there isn't a schemas folder now, but shouldn't there be?
> 
> Looks like the folder hasn’t been checked in because its being ignored even though it isn't listed in gitignore

Found it. Ill add the folder and gitignore change now

**Review by LJBabbage — APPROVED** [review]

**Review by MebinAbraham — COMMENTED** [review]
Whoops forgot to post this.

File: README.md [hunk 1]
```diff
@@ -1,38 +1,19 @@
...
-
-Remove the `/schemas` directory if it already exists
-
-Create a symbolic link pointing at your generated schema files by running the following command:
+For example in your local eq-questionnaire-runner repository, running the following command will create a symbolic link between the business folder here and the schemas directory in runner.
 ```
-ln -s <PATH_TO_REPO>/eq-questionnaire-schemas/schemas <PATH_TO_REPO>/eq-questionnaire-runner/schemas
+ln -s <PATH_TO_REPO>/eq-questionnaire-schemas/schemas/business <PATH_TO_REPO>/eq-questionnaire-runner/schemas
 ```
 You should now be able launch a questionnaire using one of the schemas.
```
Comment (MebinAbraham) [inline README.md:17]:
```suggestion
You should now be able to launch a questionnaire using one of the schemas.
```

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**